StripeConnect/StripeConnect/Source/Internal/Webview/ConnectComponentWebViewController.swift

```diff
@@ -123,19 +140,41 @@ class ConnectComponentWebViewController: ConnectWebViewController {
         }
     }
 
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        analyticsClient.logComponentViewed(viewedAt: .now)
+    }
+
     // MARK: - ConnectWebViewController
 
+    override func webView(_ webView: WKWebView, didFinish navigation: WKNavigation!) {
+        super.webView(webView, didFinish: navigation)
+
+        analyticsClient.logComponentWebPageLoaded(loadEnd: .now)
```

Comment:
Do we not want to log url here as well?

Reply:
I don't think so – it's assumed that this is the component wrapper page. If it's not, then we have a bug where we've navigated away from the component page.

Reply:
See #4252